src/components/Layout/Chat.tsx

```diff
@@ -82,7 +82,16 @@ const Chat: React.FC = () => {
           open={menuOpen}
           locked={menuLocked}
           onToggle={toggleMenuOpen}
-          className={clsx("layout_chat__menu", { "layout_chat__menu--locked": menuLocked })}
+          className={clsx(
+            "layout_chat__menu",
+            {
+              "layout_chat__menu--threadclosed": !mainActiveTopic,
+              "layout_chat__menu--threadopen": mainActiveTopic,
+            },
+            {
+              "layout_chat__menu--locked": menuLocked,
+            }
+          )}
```

Comment:
`menuLocked` is the same as `!mainActiveTopic` (https://github.com/MemeLabs/strims/blob/main/src/components/Layout/Chat.tsx#L47) so defining more modifiers for this doesn't add anything... this can all be reverted